ospf6d/ospf6_top.c

```diff
@@ -1711,6 +2118,41 @@ static int ospf6_distance_config_write(struct vty *vty, struct ospf6 *ospf6)
 	return 0;
 }
 
+static int ospf6_asbr_summary_config_write(struct vty *vty, struct ospf6 *ospf6)
+{
+	struct route_node *rn;
+	struct ospf6_external_aggr_rt *aggr;
+	char buf[PREFIX2STR_BUFFER];
+
+	if (ospf6->aggr_delay_interval != OSPF6_EXTL_AGGR_DEFAULT_DELAY)
+		vty_out(vty, " aggregation timer %u\n",
+				ospf6->aggr_delay_interval);
+
+	/* print 'summary-address A:B::C:D/M' */
+	for (rn = route_top(ospf6->rt_aggr_tbl); rn; rn = route_next(rn))
+		if (rn->info) {
+			aggr = rn->info;
+			prefix2str(&aggr->p, buf, sizeof(buf));
+			vty_out(vty, " summary-address %s ", buf);
```

Comment:
Please remove the space at the of this string, otherwise the `show running` will display:

```
 summary-address X:X::X:X  metric Y
                          ^
                          |
                     extra space
```

The space at the end of other options should also be removed in case they are used in combination.